src/components/Textarea/README.md

```diff
@@ -48,17 +90,105 @@ export default {
 </script>
 ```
 
+### Resizeable
+
+```vue
+<template>
+	<m-textarea
+		resizable
+		placeholder="Placeholder"
+	/>
+</template>
+
+<script>
+import { MTextarea } from '@square/maker/components/Textarea';
+
+export default {
+	components: {
+		MTextarea,
+	},
+};
+</script>
+```
+
+### Outline Variant
+
```

Comment:
Will these docs include any design guidance such as when this variant should be used vs the default? Or does that live elsewhere?

Reply:
Nope. Design has not provided any such guidance. Right now the guidance is "if a designer tells you to use the outline variant then use the outline variant".